src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Mod_Float.txt

```diff
@@ -0,0 +1,43 @@
+#SETUP: NumberIsFloat
+
+>> Mod(1E+307, 3)
+1
+
+>> Mod(1E+307, -3)
+-2
+
+>> Mod(-1E+307, 3)
+2
+
+>> Mod(-1E+307, -3)
+-1
+
+>> Mod(1.7976931348623157E+300, 3)
+1
+
+>> Mod(1E+305, 123)
+112
+
+>> Mod(1.2345E+260, 67.89)
+41.28195726574958
+
+>> Mod(1.7976931348623157E+308, 25)
+18
```

Comment:
This is also wrong - the result will be 0 (since the dividend is a multiple of 100). Like the previous one, we should not "spec" an incorrect result.

Reply:
removed this as well.